core/systems/input/handheld_gamepad.gd

```diff
@@ -202,12 +204,34 @@ func _emit_events(event_list: Array[InputDeviceEvent], delta: float, do_release
 			value = 0
 #		logger.debug("Emit event:" + str(event.type) + " code: "  + str(event.code) + " value: "  + str(value))
 		if _emit_event(event.get_type(), event.get_code(), value) == ERR_PARAMETER_RANGE_ERROR:
-			gamepad_device.translate_event(event, delta)
+			_do_audio_event(event, delta)
 		if event_list.size() > 1:
 			OS.delay_msec(80)
 		_emit_event(InputDeviceEvent.EV_SYN, InputDeviceEvent.SYN_REPORT, 0)
 
 
+## Translates the given event to an AudioManager event.
+func _do_audio_event(event: InputDeviceEvent, delta: float) -> void:
+	var current_volume := AudioManager.get_current_volume()
+	match event.get_code():
+		113: # EV_KEY KEY_MUTE
```

Comment:
Instead of hard-coding the numbers, can you use the constants from the InputDeviceEvent?

E.g.
```gdscript
match event.get_code():
    event.KEY_MUTE:
        ...

```

Reply:
I had to make some changes where event was no longer passed in full form to _do_audio_event, so I opted for InputDeviceEvent.KEY_...